runtime.md

```diff
@@ -103,6 +103,7 @@ This operation MUST generate an error if it is not provided the container ID.
 Attempting to start a container that does not exist MUST generate an error.
 Attempting to start an already started container MUST have no effect on the container and MUST generate an error.
 This operation MUST run the user-specified code as specified by [`process`](config.md#process-configuration).
+If `process.args` was not configured, the runtime MUST generate an error.
```

Comment:
> maybe `This operation MUST generate an error if process.args is not set`

Done (and rebased onto master) with 35ee63b → 7071706, although I used “was” instead of “is” because what matters is [what was in the configuration at `create`-time][1].

[1]: https://github.com/opencontainers/runtime-spec/blame/v1.0.0-rc3/runtime.md#L97